e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "build",
     "baseUrl": "./",
-    "types": ["cypress", "@testing-library/cypress"],
+    "types": ["cypress", "@testing-library/cypress", "cypress-real-events"],
```

Comment:
We didn't include the type from this lib as per the library's [installation guide](https://github.com/dmtrKovalenko/cypress-real-events#installation).

This change makes sure we have autocompletion for commands from `cypress-real-events` e.g. `realHover`

#### Before
![Screenshot 2023-08-22 at 5 24 40 PM](https://github.com/metabase/metabase/assets/1937582/dbd7921f-2ede-4937-8ee4-bd4c435a524a)

####  After
![Screenshot 2023-08-22 at 5 24 30 PM](https://github.com/metabase/metabase/assets/1937582/88322ff6-9ba3-437a-b8a6-2cc29f3db2f3)